### ISSUE TITLE
Entities without devices

### ISSUE THREAD

**Issue by joadske** [icanos/hassio-plejd#28]
Hi, first I want to thank you for this "add-on"!
But I have a question about "Entities without devices", everything works as it should, I can turn on/off light dim etc. but my Plejd units do not show up as "Entities without devices"
Is that how it should work, or is it possible to make them devices?

/J

**Comment by icanos** [comment]
Thank you :-)
Not sure what that means, can you elaborate please?

**Comment by joadske** [comment]
I hardly know myself ;)
But when you do, for example, an "Automation" it is much easier if it is a "Device" as there are defined "triggers". (today, the Plejd units do not show up as Devices)
No big deal, I've already solved all my automation with "mqtt.publish".

PS, I got this in the log today: (After a restart of Plejd add-on, everything works fine again.)
_(node:810) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 write listeners added. Use emitter.setMaxListeners() to increase limit
_

Again, a big thank you for your job.
